amp/repository/um/view/viewEditUser.jsp

```diff
@@ -1,677 +1,677 @@
+																	    </td>
+																	</tr>
+																	<tr>
+																		<td width="169" align="right" height="30"style="font-size: 11px;
+    font-weight: bold; color:#000;">
+																	        <digi:trn key="aim:viewEditUser:email">Email </digi:trn>
+																	        <font color="red">*</font>
+																		</td>
+																		<td width="190" height="30">
+																           <html:text name="umViewEditUserForm"
+																					  property="email" styleClass="pwd_username" style="background-color: #FFFFFF;border: 1px solid #D0D0D0;color: #767676;font-size: 11px;margin: 5px;padding: 2px; width:180px;" styleId="userMail"/>
+																		</td>
+																	</tr>
+																	<tr>
+                                                                        <td width="169" align="right" height="30"style="font-size: 11px; font-weight: bold; color:#000;">
+                                                                             <digi:trn>Use different email for email notifications</digi:trn>
+                                                                        </td>
+                                                                        <td width="380" height="30" colspan="2">
+                                                                            <html:checkbox property="notificationEmailEnabled" styleClass="inp-text" style="margin: 5px" styleId="notificationEmailEnabled"/>
+                                                                        </td>
+                                                                    </tr>
+	                                                                    <tr id="notificationEmailRow">
+	                                                                        <td width="169" align="right" height="30"style="font-size: 11px; font-weight: bold; color:#000;">
+	                                                                            <digi:trn>Notification Email</digi:trn>
+	                                                                            <font color="red">*</font>
+	                                                                        </td>
+	                                                                        <td width="190" height="30">
+	                                                                           <html:text name="umViewEditUserForm" property="notificationEmail" style="background-color: #FFFFFF;border: 1px solid #D0D0D0;color: #767676;font-size: 11px;margin: 5px;padding: 2px; width:180px;" styleId="notificationEmail"/>
+	                                                                        </td>
+	                                                                    </tr>
+																	<tr>
```

Comment:
line 154 has a message that needs to be translated

Reply:
"Please Select Country" is a translatable message. 
On the other hand, the 'country' in this context is not related to adm levels. No action is needed here.